app/services/i18n.js

```diff
@@ -301,35 +267,33 @@ var I18nService = Ember.Service.extend({
   _initLibrary: function () {
     var self = this;
 
-    var isThennable = function (obj) {
-      return obj && obj.then && typeof obj.then === 'function';
-    };
-
     return new Ember.RSVP.Promise(function (resolve, reject) {
       var i18next = self.get('i18next');
       var options = config.i18nextOptions || {};
 
-      var initResponse = i18next.init(options);
-
-      if (isThennable(initResponse)) {
-        initResponse.then(function () {
-          resolve(i18next);
-        }, function (reason) {
-          reject(reason);
-        });
-      } else {
-        Ember.warn('The response from i18next.init() was not a promise.');
-        resolve(i18next);
-      }
+      //
+      //  TODO:  Adding i18nextXHRBackend by default so that translation
+      //         files can be loaded.
+      //         How do we extend this so that other plugins can be added
+      //         dynamically?
+      //
+      i18next
+        .use(window.i18nextXHRBackend)
+        .init(options, (err, t) => {
```

Comment:
Please remove the extra spaces after the parens below to match the rest of the code.